pom.xml

```diff
@@ -1320,8 +1321,21 @@
 
             <dependency>
                 <groupId>org.mockito</groupId>
-                <artifactId>mockito-all</artifactId>
+                <artifactId>mockito-core</artifactId>
```

Comment:
re-add mockito-all just here, as wildfly full needs it and  build fails if not defined here.
Keep both defined here for now, but add comment on why the --all is still here.